src/runtime/wasm.ts

```diff
@@ -394,27 +449,26 @@ export class WasmParticle extends Particle {
   private store(str: string): WasmAddress {
     const p = this.exports._malloc(str.length + 1);
     for (let i = 0; i < str.length; i++) {
-      this.heap[p + i] = str.charCodeAt(i);
+      this.heapU8[p + i] = str.charCodeAt(i);
```

Comment:
Should we add some kind of assert to check for charCodes > 255?

Reply:
I don't mind one way or the other. It's probably going to be quite some time before anyone tries to use unicode here.